app/serializers/teacher_set_serializer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class TeacherSetSerializer < ActiveModel::Serializer
-  cached
+  # cached
```

Comment:
I'd like to understand why caching is commented out.

Reply:
I need to research on this. If 'cached' varibale is enabled data is not working.